Makefile

```diff
@@ -727,8 +727,13 @@ test-integration: ## run all integration tests.
 		CNS_VERSION=$(CNS_VERSION) \
 		go test -mod=readonly -buildvcs=false -timeout 1h -coverpkg=./... -race -covermode atomic -coverprofile=coverage.out -tags=integration ./test/integration...
 
+test-load: ## run all load tests
```

Comment:
Are we using this make command anywhere ?

Reply:
Yeah in the next PR